pkg/cloud/libvirt/client/domain.go

```diff
@@ -108,7 +108,8 @@ func newDevicesDef(virConn *libvirt.Connect) *libvirtxml.DomainDeviceList {
 	}
 	// Both "s390" and "s390x" are linux kernel architectures for Linux on IBM z Systems, and they are for 31-bit and 64-bit respectively.
 	// Graphics/Spice isn't supported on s390/s390x platform.
-	if arch != "s390x" && arch != "s390" {
+	// Same case for PowerPC systems as well
+	if !strings.HasPrefix(arch, "s390") && !strings.HasPrefix(arch, "ppc64") {
```

Comment:
Fine with me but spice change is a separate logical change so please split the commit (not the PR). Pro tip: If you need to use the word "also" in your commit log, it usually means you've more than one change in your patch. :)

Reply:
thanks @zeenix . got it. I was just being lazy ;)

Reply:
opened #186

Reply:
Happens to all of us. Please note that I only asked for the commits to be split (not the PR). It's not a problem at all and some might argue it's even better, just for future reference. :)